assets/src/styles/blocks/Covers/layouts/CoversCarouselLayout.scss

```diff
@@ -0,0 +1,97 @@
+// Carousel layout
+.covers-block.carousel-layout {
+  .covers-container {
+    position: relative;
+  }
+
+  .covers {
+    display: flex;
+    overflow-x: scroll;
+    flex-wrap: nowrap;
+    -ms-overflow-style: none;
+    scrollbar-width: none;
+    scroll-behavior: smooth;
+
+    &::-webkit-scrollbar {
```

Comment:
Why Webpack doesn't solve this workaround?

Reply:
I'm not sure... But if I remove this line then the scrollbar is shown 😕